yuca/utils.py

```diff
@@ -84,7 +84,7 @@ def prep_input(img, external_channels=None, \
 
     return batch
 
-def make_target(img, batch_size):
+def make_target(img):
```

Comment:
removed erroneous batch_size arg. 
I grep'ed for this pattern in other modules and didn't find anywhere this might break anything, and tests all still pass.
Nothing broke, but that does raise the question of whether this is a useful function.